netty/src/main/java/io/grpc/netty/NettyClientHandler.java

```diff
@@ -445,12 +445,7 @@ public void handleProtocolNegotiationCompleted(
     this.attributes = this.attributes.toBuilder().setAll(attributes).build();
     this.securityInfo = securityInfo;
     super.handleProtocolNegotiationCompleted(attributes, securityInfo);
-    // Once protocol negotiator is complete, release all writes and remove the buffer.
-    ChannelHandlerContext handlerCtx =
-        ctx().pipeline().context(WriteBufferingAndExceptionHandler.class);
-    if (handlerCtx != null) {
-      ((WriteBufferingAndExceptionHandler) handlerCtx.handler()).writeBufferedAndRemove(handlerCtx);
-    }
+    InternalWriteBufferingAndExceptionHandlerUtils.writeBufferingAndRemove(ctx().channel());
```

Comment:
nit: This cannot call into an internal package.   Youll need to expose writeBufferingAndRemove as a package-private static on the NCH itself, and make this call into NCH

Reply:
done.